docs/index.yml

```diff
@@ -35,9 +35,9 @@ highlightedContent:
       title: "Microsoft Learn: Extend Office clients with Office Add-ins"
       url: /learn/paths/m365-office-add-in-associate
     # Card 5
-    - itemType: whats-new
-      title: Microsoft 365 Developer Blog
-      url: https://developer.microsoft.com/office/blogs/technology/office-add-ins
+    - itemType: sample
+      title: "Samples for Office Add-ins"
```

Comment:
Not sure it matters, but why is this in quotation marks?

Reply:
It ensures the YAML is interpreted correctly as a string and various characters are not parsed incorrectly as YAML characters. Might be fine in this case, but when in doubt, quotes don't hurt anything.